nodes/CalDavNode/CalDavNode.node.ts

```diff
@@ -0,0 +1,101 @@
+import {
+	IDataObject,
+	IExecuteFunctions,
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription, NodeOperationError,
+} from 'n8n-workflow';
+import {calendar, event} from "./actions";
+import {loadOptions} from "./methods";
+import {CalDav} from "./actions/Interface";
+
+export class CalDavNode implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'CalDAV Node',
+		name: 'calDavNode',
+		icon: 'file:CalDavNode.svg',
```

Comment:
I noticed, that the file starts with a lowercase `c` – not sure if that is important, but I think it makes sense to have it consistent.

Reply:
File on my Mac starts with capital `C`. Committed with `git config core.ignorecase false`. Now config matches filename.